igor-web/src/main/java/com/netflix/spinnaker/igor/helm/model/HelmEntry.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Apple, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.helm.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class HelmEntry {
+  public String apiVersion;
```

Comment:
Another comment: I think you only need to define the fields that are actually used by Igor here. In that way we could prevent tons of yaml unmarshall errors when using different helm repo implementations and the index fields change.

Reply:
These fields have been pruned to the minimum set.